src/pages/AboutView/AboutView.js

```diff
@@ -0,0 +1,71 @@
+import screenshot from './desktop-screen.png';
+import './about-view.css';
+import { NavLink } from 'react-router-dom';
+const { Helmet } = require('react-helmet');
+
+const AboutView = ({ listId }) => {
+  return (
+    <div className="page_view_about page-view ">
+      <Helmet>
+        <title>About - Peasy</title>
+      </Helmet>
+      <div className="about-images">
+        <img src={screenshot} alt="desktop view" className="desktop__image" />
+      </div>
+      <main className="about-text">
+        <h1 className="about-text__header">How Peasy Works</h1>
+        <section className="about-section">
+          <h2 className="about-section__header">
+            Add items to your grocery list
+          </h2>
+          <p className="about-section__paragraph">
+            Add items to your grocery list and select how soon you'll need to
+            buy your next shopping item.
+          </p>
+        </section>
+        <section className="about-section">
+          <h2 className="about-section__header">Check items off your list</h2>
+          <p className="about-section__paragraph">
+            As you buy your groceries, check them off your grocery list.
+          </p>
+        </section>
+        <section className="about-section">
+          <h2 className="about-section__header">
+            Predict when you'll buy next
```

Comment:
I think this shouldn't read as imperitive because it's confusing, and makes it sound like something the user needs to do. Hmmm, how about:
```suggestion
            Peasy predicts what you'll need to buy next
```
or something similar?